pkg/profiles/profiles.go

```diff
@@ -43,8 +44,18 @@ var (
 		Routes:      []*pb.Route{},
 		RetryBudget: &DefaultRetryBudget,
 	}
+	// DefaultRouteTimeout is the default timeout for routes that do not specify
+	// one.
+	DefaultRouteTimeout = 1 * time.Second
```

Comment:
Before we removed the bind timeout in #2017, we effectively had a default timeout of 10 seconds. I'm a bit worried that setting a default timeout of 1 second will be too aggressive for some installations, especially those that were previously reliant on the 10 second time. How would you feel about setting this default to 10 seconds to match?